packages-native/actions/src/camera/TakePicture.ts

```diff
@@ -91,10 +89,8 @@ function TakePicture(
                     const guid = imageObject.getGuid();
                     // eslint-disable-next-line no-useless-escape
                     const filename = /[^\/]*$/.exec(uri)![0];
-                    // eslint-disable-next-line @typescript-eslint/explicit-function-return-type
-                    const onSuccess = () => resolve(true);
-                    // eslint-disable-next-line @typescript-eslint/explicit-function-return-type
-                    const onError = (error: Error) => reject(error.message);
+                    const onSuccess = (): void => resolve(true);
```

Comment:
do we ever return `false`, expect that when canceled? if not then the return type should be void